app/assets/scss/application.scss

```diff
@@ -1,82 +1,55 @@
-// GOVUK Front-end toolkit styles
-@import "_grid_layout.scss";
-@import "_typography.scss";
+// GOVUK Front-end toolkit variables, mixins and functions
+@import "_grid_layout.scss";    // Grid variables
+@import "_typography.scss";     // Core bold and heading mixins
 @import "_url-helpers.scss";
 
 // Path to assets for use with file-url()
 $path: "/static/images/";
 
-// Digital Marketplace front end toolkit components
-@import "forms/word-counter";
-@import "forms/list-entry";
-
-/* Classes shared between multiple apps */
-@import "toolkit/shared_scss/_reset.scss";
-@import "toolkit/shared_scss/_lists.scss";
-@import "toolkit/shared_scss/_dmspeak.scss";
-
-/* Blocks shared between multiple selectors */
+// Digital Marketplace Front-end toolkit shared CSS placeholders used by other DM toolkit styles
 @import "toolkit/shared_placeholders/_temporary-messages.scss";
 @import "toolkit/shared_placeholders/_placeholders.scss";
 @import "toolkit/shared_placeholders/_dm-typography.scss";
 @import "toolkit/shared_placeholders/_mixins.scss";
 
-// Digital Marketplace Front-end toolkit styles
+// Digital Marketplace Front-end toolkit shared styles between multiple apps
+@import "toolkit/shared_scss/_reset.scss";
+@import "toolkit/shared_scss/_lists.scss";
+@import "toolkit/shared_scss/_dmspeak.scss";
+
+// Digital Marketplace Front-end toolkit styles (only import the files you need)
 @import "toolkit/_breadcrumb.scss";
-@import "toolkit/_browse-list.scss";
 @import "toolkit/_buttons.scss";
-@import "toolkit/_contact-details.scss";
 @import "toolkit/_document.scss";
 @import "toolkit/_grids.scss";
 @import "toolkit/_instruction-list.scss";
-@import "toolkit/_link-button.scss";
 @import "toolkit/_notification-banners.scss";
 @import "toolkit/_page-headings.scss";
 @import "toolkit/_phase-banner.scss";
 @import "toolkit/_proposition-header.scss";
 @import "toolkit/_secondary-action-link.scss";
-@import "toolkit/_service-id.scss";
-@import "toolkit/_temporary-message.scss";
 @import "toolkit/forms/_dates.scss";
 @import "toolkit/forms/_hint.scss";
-@import "toolkit/forms/_keyword-search.scss";
-@import "toolkit/forms/_option-select.scss";
 @import "toolkit/forms/_questions.scss";
 @import "toolkit/forms/_selection-buttons.scss";
 @import "toolkit/forms/_summary.scss";
 @import "toolkit/forms/_textboxes.scss";
 @import "toolkit/forms/_validation.scss";
-@import "toolkit/search/_search-result.scss";
+@import "toolkit/forms/list-entry";
```

Comment:
Can we do these consistently with the other tookit imports (adding a preceding underscore and the file extension). We could also order alphabetically too

Reply:
👍 to consistency and alphabetisation.

Reply:
I have broken things alphabetising them in the past - import order can matter for scss. (I have no idea of its a potential issue here though.)

Reply:
Hopefully we will pick up if something has broken. I think in the long run we should be able to structure our toolkit and scss files such that alphabetical ordering should not cause any problems (our files should not have unexpected knock on effects for other components). In the meantime if we spot anything explicitly depending on non alphabetical order let's add it as an inline comment